tests/test_things.py

```diff
@@ -412,6 +412,10 @@ def test_thingsdate(self):
         self.assertEqual("AND stopDate IS NULL", sqlfilter)
 
 
+    def test_thingstime(self):
+        test_task = things.tasks("7F4vqUNiTvGKaCUfv5pqYG")['reminder_time']
```

Comment:
You might want to do a `.get("reminder_time")` here, since the key may be missing in erroneous situations.